src/wallet/CarbonWallet.ts

```diff
@@ -426,6 +426,7 @@ export class CarbonWallet {
           handler: { resolve, reject },
         });
       } else {
+        console.log({error})
         reject(error);
       }
     }
```

Comment:
Don't forget to update the amino types for the txs you have added as well.
<img width="808" alt="Screenshot 2023-02-10 at 2 06 42 PM" src="https://user-images.githubusercontent.com/65583522/218015643-a531f156-fb53-4203-910c-573e4b5c70cd.png">

You can find these under `x/coin/types/codec.go` and `x/cdp/types/codec.go`